### PR TITLE
Update description

### PR DESCRIPTION
**Pull request by ethomson**

### CODE REVIEW

**Comment by IAmHughes** [comment]
@ethomson for this to be approved, we need to also tag the latest version of the action to match this.

File: README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ jobs:
           prerelease: false
       - name: Upload Release Asset
         id: upload-release-asset 
-        uses: actions/upload-release-asset@v1.0.1
+        uses: actions/upload-release-asset@v1
```
Comment (birtles) [inline README.md]: For what it's worth, this change didn't work for me. I get:
```
Download action repository 'actions/upload-release-asset@v1'
##[warning]Failed to download action 'https://api.github.com/repos/actions/upload-release-asset/tarball/v1'. Error Response status code does not indicate success: 404 (Not Found).
```

**Comment by derTobsch** [comment]
Now that the tags are set (see https://github.com/actions/upload-release-asset/releases) this is able to merge or @IAmHughes?

File: README.md [hunk 2]
```diff
@@ -36,13 +36,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
-        uses: actions/checkout@master
+        uses: actions/checkout@v1
```
Comment (derTobsch) [inline README.md]: now you can go to v2 :)

Comment (IAmHughes) [inline README.md]:
```suggestion
        uses: actions/checkout@v2
```

Comment (IAmHughes) [reply README.md]: making change, thanks!

File: README.md (hunk 1 above)
Comment (IAmHughes) [reply README.md]: This should be resolved now

**Pull request merged**